### PR TITLE
Feature/7027 custom field enhancement

### PR DESCRIPTION
**Pull request by Anmol79**
http://pm.7vals.com/issues/7027

### CODE REVIEW

**Comment by Anmol79** [comment]
Cycle 0

File: app/models/custom_attributes/boolean_value.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module CustomAttributes
+  module BooleanValue
+    def value=(newVal)
+      if !newVal.nil?
+        self.boolean_value = newVal
+      else
+        self.boolean_value = nil
+      end
```
Comment (kashif-umair) [inline app/models/custom_attributes/boolean_value.rb]: why not just write this single line?
```ruby
self.boolean_value = newVal
```

File: app/views/custom_attributes/boolean_values/_edit_boolean_value.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      <%= custom_fields.label boolean_value.name.to_sym, label_text.html_safe, class: cssClasses[:title] %>
+      <%= custom_fields.radio_button "#{boolean_value.name}".to_sym, true, checked: boolean_value.boolean_value, class: "#{cssClasses[:field]}", required: true %> Yes
+      <br />
+      <%= custom_fields.radio_button "#{boolean_value.name}".to_sym, false, checked: !boolean_value.boolean_value.nil? && !boolean_value.boolean_value, class: "#{cssClasses[:field]}", required: true %> No<br />
+    <% end %>
+  <% # END Field part of form %>
+<% else %>
+  <% # BEGIN Inline Editing %>
+  <div class="custom-attributes editable-inline" data-controller="<%= boolean_value.owner.class.to_s.underscore %>_custom_attribute_values" data-owner-id="<%= boolean_value.owner.id %>" data-attr-name="<%= boolean_value.name %>">
+    <div class="<%=cssClasses[:title]%>">
```
Comment (kashif-umair) [inline app/views/custom_attributes/boolean_values/_edit_boolean_value.html.erb]: spacing.

**Comment by kashif-umair** [comment]
Cycle 1

**Pull request merged**